modules/openapi-generator/src/main/resources/dart/libraries/dio/auth/basic_auth.mustache

```diff
@@ -1,33 +1,34 @@
 {{>header}}
-import 'dart:async';
 import 'dart:convert';
 
 import 'package:dio/dio.dart';
 import 'package:{{pubName}}/src/auth/auth.dart';
 
 class BasicAuthInfo {
-    final String username;
-    final String password;
+  final String username;
+  final String password;
 
-    const BasicAuthInfo(this.username, this.password);
+  const BasicAuthInfo(this.username, this.password);
 }
 
 class BasicAuthInterceptor extends AuthInterceptor {
-    Map<String, BasicAuthInfo> authInfo = {};
+  Map<String, BasicAuthInfo> authInfo = {};
```

Comment:
final for consistency?

Reply:
Sure, this was just a copy/paste job from dart-dio.